olp-cpp-sdk-dataservice-read/src/generated/api/BlobApi.h

```diff
@@ -23,7 +23,9 @@
 
 #include <olp/core/client/ApiError.h>
 #include <olp/core/client/ApiResponse.h>
+#include <olp/core/client/HttpResponse.h>
 #include <boost/optional.hpp>
+#include "../../ExtendedApiResponse.h"
```

Comment:
```suggestion
#include "ExtendedApiResponse.h"
```
I believe, olp-cpp-sdk-dataservice-read/src folder added to include directories.